examples/petsc/bp3.cu

```diff
@@ -88,5 +88,4 @@ extern "C" __global__ void Error(void *ctx, CeedInt Q,
        i += blockDim.x * gridDim.x) {
     err[i] = u[i] - target[i];
   }
-  return 0;
```

Comment:
Eventually the goal is to have all QFunctions return an error code (OCCA, CUDA, and C) as we want the user to be able to add diagnostics to their QFunctions to indicate if something goes awry.

Reply:
__global__ functions have to be void.

Reply:
I remember some discussion at CEED2AM about how to get it a return value to work for OCCA, but I don't recall all of it. I just recall the desire for the user to set return values.